iohk/NixOps.hs

```diff
@@ -348,28 +349,34 @@ readConfig cf = do
     die $ format ("Config file '"%fp%"' is incoherent with respect to elements "%w%":\n  - stored files:  "%w%"\n  - implied files: "%w%"\n")
           cf cElements (sort cFiles) (sort deducedFiles)
   pure c
-  
+
 
 parallelIO :: Options -> [IO a] -> IO ()
 parallelIO Options{..} =
   if oSerial
   then sequence_
   else sh . parallel
 
+logCmd  cmd args = do
+  printf ("-- "%s%"\n") $ T.intercalate " " $ cmd:args
+  Sys.hFlush Sys.stdout
+
+inproc :: Text -> [Text] -> Shell Line -> Shell Line
```

Comment:
It might not be in this branch, but might be in other -- it's sort of a symmetric element in a diamond of various options of calling shell stuff.